src/app/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.ts

```diff
@@ -16,18 +14,85 @@ export class UbsAdminOrderDetailsFormComponent implements OnInit, OnDestroy {
   public isInputDisabled = false;
   public isVisible = true;
   public ubsCourier = 0;
+  private order;
+  public orderInfo = {
+    amount: {
+      planned: 0,
+      confirmed: 0,
+      actual: 0
+    },
+    sum: {
+      planned: 0,
+      confirmed: 0,
+      actual: 0
+    },
+    finalSum: {
+      planned: 0,
+      confirmed: 0,
+      actual: 0
+    },
+    bonuses: 0,
+    certificateDiscount: 0
+  };
   public orderDetailsForm: FormGroup;
```

Comment:
can we use general "orderForm" from ubs-admin-order-component to manage all data together?

Reply:
done